src/modelstats.jl

```diff
@@ -3,47 +3,18 @@ export dic
 #################### Posterior Statistics ####################
 
 """
-    dic(chain::Chains, logpdf::Function) -> (DIC, pD)
+    dic(chain::Chains)
 
-Compute the deviance information criterion (DIC).
-(Smaller is better)
+Computes Deviance Information Criterion (DIC).
```

Comment:
The first one-line description of the method [should use the imperative form](https://docs.julialang.org/en/v1/manual/documentation/):
```suggestion
Compute the deviance information criterion (DIC).
```